package.json

```diff
@@ -1,10 +1,18 @@
 {
   "private": true,
   "scripts": {
-    "build:css": "sass sass:css",
-    "watch:css": "sass -w sass:css"
+    "watch": "NODE_ENV=production postcss ./sass/*.scss -d ./css --ext css -w",
+    "css": "NODE_ENV=production postcss ./sass/*.scss -d ./css --ext css"
```

Comment:
Can we go with something like this without the the `NODE_ENV` and move `sass` to `src/css`?
```
    "build:css": "postcss src/css/*.css -o css/main.css",
    "watch:css": "postcss src/css/*.css -o css/main.css --watch"
```